engine/api/src/main/java/org/apache/cloudstack/engine/subsystem/api/storage/DataMotionStrategy.java

```diff
@@ -25,11 +25,28 @@
 import com.cloud.agent.api.to.VirtualMachineTO;
 import com.cloud.host.Host;
 
+/**
+ * interface to query how to move data around and to commision the moving
```

Comment:
I am loving this new approach that you are adopting to document the code you introduce in ACS 👍

Reply:
yes, i was feeling preppy ;) and in this case, I think it does say something more than the name implies.

Reply:
done